src/definitions.jl

```diff
@@ -33,11 +41,11 @@ realfloat(x::AbstractArray{T}) where {T<:Real} = copy1(typeof(fftfloat(zero(T)))
 
 # copy to a 1-based array, using circular permutation
 function copy1(::Type{T}, x) where T
-    y = Array{T}(map(length, indices(x)))
+    y = Array{T}(uninitialized, map(length, axes(x)))
```

Comment:
I think this will need to be `Compat.axes` for 0.6 compatibility. (It had to be unexported from Compat to avoid breaking AxisArrays et al.)

Reply:
Ah, yes, of course ... except that it passed CI, although the used Compat 0.46 should not be exporting it. 😕 
Will change anyway.

Reply:
To clarify: This file isn't even loaded on 0.6.